scripts/asm-installer/asm_vm

```diff
@@ -1110,6 +1110,23 @@ this project, run:
   --project="${PROJECT_ID}"
 EOF
     fi
+
+    info "Verifying GCE image..."
+    local SOURCE_IMAGE_PATH="$(echo "$(retry 2 run gcloud compute instance-templates \
+      describe "${SOURCE_INSTANCE_TEMPLATE}" \
+      --format=json \
+      --project="${PROJECT_ID}")" \
+    | jq '.properties.disks[] | select(.initializeParams.sourceImage != null) | .initializeParams.sourceImage')"
+
+
+    if [[ "$SOURCE_IMAGE_PATH" == *projects/*-cloud/global/images*  ]]; then
+    # check whether the public image is supported
+      if [[ "$SOURCE_IMAGE_PATH" != *debian-9* ]] && [[ "$SOURCE_IMAGE_PATH" != *debian-10* ]]; then
+        fatal "Image not supported. Supported images are: debian9, debian10"
```

Comment:
I recommend putting support images in a list and each time we add a supported image, we can just add to that list instead of adding in the actual logic.

Reply:
added